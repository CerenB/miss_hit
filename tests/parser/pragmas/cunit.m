% (c) Copyright 2020 Florian Schanda

% This pragma applies to the entire compilation unit
% mh:debug_pragma

classdef cunit
end
