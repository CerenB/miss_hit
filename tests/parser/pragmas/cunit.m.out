----------------------------------------------------------------------
--  Parse tree for cunit.m
Root: Class_File
  Pragmas: Simple_Pragma debug_pragma <None>
  Classdef: Class_Definition
    Name: Identifier <cunit>
----------------------------------------------------------------------
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
